src/actions/actions.ts

```diff
@@ -413,7 +413,7 @@ class CommandEsc extends BaseCommand {
     ModeName.SearchInProgressMode,
     ModeName.Replace
   ];
-  keys = ["<escape>"];
+  keys = ["<Esc>"];
```

Comment:
This must have been annoying to do. Thanks.

I would love to extract these out into a string enum at some point.